### DISCUSSION TITLE
Asendia Shipments marked as Delivered Immediately in Staging

### DISCUSSION THREAD

**Discussion by wesbos** [chitchats/chitchats-api-doc#38] [Q&A]
Looks like Asendia shipments are marked as delivered immedaitely, which is an issue when I'm trying to add those shipments to a batch in staging mode and integrate into my selling platform.

I assume this doesn't happen in production, but it's difficult to test Asendia shipments now. Is this a known issue? 

<img width="673" alt="image" src="https://user-images.githubusercontent.com/176013/108122007-a8c48980-7071-11eb-8c28-86ef2430798e.png">

**Comment by gshaw** [comment]
Thanks for pointing this out.  This doesn't happen on production, in fact sometimes the opposite happens with Asendia tracking but I digress... I'll get this investigated and comment back here when I have something to share.

**Comment by gshaw** [comment]
@wesbos sorry for the late reply.  I confirmed that the shipment will get tracking events and resolve itself as delivered 1 hour after buying the postage.  The tracking events will be all over the place, we don't have control over them but we delay updating shipments on staging for 1 hour to give you a chance to test them.

Hopefully that helps you understand what is going on and you'll be able to work within that delay.

**Reply by wesbos** [reply]
Makes sense. thanks again for the help!
